worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R3/regen/PaperweightRegen.java

```diff
@@ -273,9 +273,6 @@ protected boolean initNewWorld() throws Exception {
                             WorldEditPlugin.getInstance().getBukkitImplAdapter().getInternalBiomeId(options.getBiomeType())
                     ) : null;
 
-            @Override
```

Comment:
This method must be kept

Reply:
This method is no longer existing at folia. Thats one many reasons why we have multiple adapters and seperated from main adapter